### PR TITLE
Train hpt

### PR DESCRIPTION
**Pull request by zaitmambet**
model hyperparameter tuning

### CODE REVIEW

**Review by KernelA — APPROVED** [review]

File: configs/scheduler/multistep.yaml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 _target_: torch.optim.lr_scheduler.MultiStepLR
 milestones: [10]
-gamma: 0.1
\ No newline at end of file
+gamma: ^
```
Comment (KernelA) [inline configs/scheduler/multistep.yaml]: Тут что-то не то

**Review by KernelA — CHANGES_REQUESTED** [review]
Тут что-то не то исправилось в конфиге

**Review by KernelA — APPROVED** [review]
